Library/Homebrew/shims/mac/super/curl-config

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -p
+
+# HOMEBREW_LIBRARY is set by bin/brew
+# HOMEBREW_CURL_CONFIG is set by brew.sh
+# shellcheck disable=SC2154
+source "${HOMEBREW_LIBRARY}/Homebrew/shims/utils.sh"
+
+# Check if HOMEBREW_CURL_CONFIG is set, fallback to system curl-config if not.
+curl_config_exec="${HOMEBREW_CURL_CONFIG:-/usr/bin/curl-config}"
+
+# Intercept --features and append HTTP2 if needed
+if [[ "$1" == "--features" ]]
+then
+  output="$("${curl_config_exec}" "$@")"
+
+  # Check if HTTP2 is missing but supported
+  if [[ ! "${output}" =~ HTTP2 ]]
+  then
+    curl_version="$("${HOMEBREW_CURL:-curl}" -V)"
+    if [[ "${curl_version}" =~ HTTP/2 ]]
+    then
+      output="${output} HTTP2"
+    fi
+  fi
+
+  echo "${output}"
+  exit 0
+fi
+
+# Fall back to executing the original curl-config
+try_exec_non_system "${curl_config_exec}" "$@"
```

Comment:
Move this before the code above - it should not apply to non-system curl-config as only system curl-config is bugged.